addons/vehicleDamage/functions/fnc_injureOccupants.sqf

```diff
@@ -0,0 +1,58 @@
+#include "script_component.hpp"
+/*
+ * Author: Brandon (TCVM)
+ * Injures occupants in a vehicle based on percent chance of injury
+ *
+ * Arguments:
+ * 0: The vehicle
+ * 1: Injury Chance
+ * 2: Maximum people to injure
+ * 3: Projectile source
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [myVehicle, 0.6, 10] call ace_vehicleDamage_fnc_injureOccupants;
+ *
+ *
+ * Public: No
+ */
+params ["_vehicle", "_chance", "_count", ["_source", objNull]];
+TRACE_4("adding damage to units", _vehicle, _chance, _count, _source);
+
+private _crewCount = count crew _vehicle;
+if (_crewCount <= 0) exitWith {};
+
+private _crewInjuryIndices = [];
+for "_i" from 0 to _crewCount - 1 do {
```

Comment:
forEach over crew _vehicle (store crew in variable, as you reuse it in L44)
and then pushBack _forEachIndex -1 ?